scss/blocks/figma-components.scss

```diff
@@ -0,0 +1,101 @@
+:root {
+    --checkered-canvas-bg: #fff;
+    --checkered-square: #182d660a;
+    --checkered-square-size: 8px;
+}
+
+.figma-components {
+    display: grid;
+    gap: 24px;
+    margin: 12px 0;
+}
+
+.figma-component {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+}
+
+.figma-component-details {
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+}
+
+.figma-component-image {
+    display: flex;
+    justify-content: center;
+    padding: 8px;
+}
+
+.figma-component-image-background {
+    background-image: linear-gradient(45deg,var(--checkered-square) 25%,transparent 25%,transparent 75%,var(--checkered-square) 75%,var(--checkered-square) 100%),linear-gradient(45deg,var(--checkered-square) 25%,var(--checkered-canvas-bg) 25%,var(--checkered-canvas-bg) 75%,var(--checkered-square) 75%,var(--checkered-square) 100%);
+    background-position: 0 0,calc(var(--checkered-square-size) / 2) calc(var(--checkered-square-size) / 2);
+    background-size: var(--checkered-square-size) var(--checkered-square-size);
+}
+
+.figma-component-image-border {
+    border-radius: 5px;
+    border: 1px solid #eceef3;
+}
+
+.figma-component-image img {
+    max-height: 100%;
+    max-width: 100%;
+    object-fit: contain;
+    height: auto;
+}
+
+.figma-component-name {
+    font-size: 14px;
+    font-weight: 500;
+    line-height: 20px;
+    color: #14171F;
+}
+
+.figma-component-description {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 16px;
+    color: #606981;
+}
+
+.figma-components-canvas {
+    border: 1px solid #eceef3;
+}
+
+.figma-component-popover {
+    box-shadow: 0px 5px 10px 0px rgba(20, 23, 31, 0.05), 0px 15px 25px 0px rgba(20, 23, 31, 0.07);
+    border-radius: 8px;
+    border: 1px solid #ECEEF3;
+    background: #FFF;
+    padding: 8px;
+}
+
+.figma-component-properties {
+    display: grid;
+    align-items: center;
+    gap: 8px;
+    grid-template-columns: repeat(2, min-content);
+}
+
+.figma-component-property-name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 16px;
+    color: #14171F;
+}
+
+.figma-component-property-value {
```

Comment:
This looks the same like badge used in the table.

Reply:
[da4abd6](https://github.com/Supernova-Studio/exporter-documentation/pull/167/commits/da4abd6af34b4d0402b88945bfd0c685a36e89a7)